core/include/tangram/platform.h

```diff
@@ -45,6 +49,12 @@ class Platform {
 
     virtual bool isContinuousRendering() const;
 
+    // returns the path of an asset given the name of the asset
```

Comment:
This description line can probably be removed since it's covered under the other specific cases. We could also  replace "given the name of the asset" -> "given the path of the asset within the application bundle" for iOS and Android.